src/AutoRest.CSharp/LowLevel/Generation/DpgClientWriter.cs

```diff
@@ -603,19 +603,19 @@ private void WriteSampleRefsIfNecessary(MethodSignature methodSignature, bool is
                 return;
 
             var sampleProvider = _library.GetSampleForClient(_client);
-            // do not write this part when there is no sample provider
+            // do not write this part when there is no sample providerPara
```

Comment:
typo? maybe we need to revert this change

Reply:
yep